internal/analytics/deferred.go

```diff
@@ -44,7 +49,7 @@ func isDeferralDayAgo(cfg Configurable) bool {
 	return diff > 24*time.Hour
```

Comment:
This checks if the modification time of the deferrer log is older than 24 hours.  But as we are appending to that file, running a `state exec` command every day would ensure that this never happens.  

The simple solution I had was to have two different files: one that is only touched on the first deferral, and the log.  At least on linux, you could probably also compare against the "creation" time of that log file, but I think that is not cross-platform...